api/views/CloudRecordSummaryView.py

```diff
@@ -113,7 +124,14 @@ def get(self, request, format=None):
 
         cursor = database.cursor(MySQLdb.cursors.DictCursor)
 
-        if group_name is not None:
+        if global_user_name is not None:
```

Comment:
it now returns a 400 status code (Bad Request) if there is more than one of `user`, `group` or `service` defined. If only `from` is defined, the request is allowed to continue as that is a valid query